cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/wizard-simple/AwsManagementClusterSimple.tsx

```diff
@@ -2,14 +2,14 @@
 import React, { useContext } from 'react';
 
 // App imports
-import { AwsStore } from '../../state-management/stores/Store.aws';
-import { TAB_NAMES } from '../../shared/constants/NavRoutes.constants';
-import Wizard from '../../shared/components/wizard/Wizard';
-import ManagementClusterSettings from '../../shared/components/management-cluster-settings-step/ManagementClusterSettings';
+import { AwsStore } from '../../../../state-management/stores/Store.aws';
+import { TAB_NAMES } from '../../../../shared/constants/NavRoutes.constants';
```

Comment:
I'm not sure it really makes sense to include all the workflow tabs names in a single TAB_NAMES array. Feels like each workflow should define their own

Reply:
+1, will update this